plugin/parser/differ/mysql/column_test.go

```diff
@@ -0,0 +1,42 @@
+package mysql
+
+import (
+	"testing"
+
+	"github.com/pingcap/tidb/parser"
+	"github.com/stretchr/testify/require"
+)
+
+func TestColumn(t *testing.T) {
+	tests := []struct {
+		old  string
+		new  string
+		want string
+	}{
+		{
+			old:  `CREATE TABLE book(id INT, PRIMARY KEY(id))`,
+			new:  `CREATE TABLE book(id INT, price INT, PRIMARY KEY(id));`,
+			want: "ALTER TABLE `book` ADD COLUMN (`price` INT);\n",
+		},
+		{
+			old:  `CREATE TABLE book(id INT, PRIMARY KEY(id))`,
+			new:  `CREATE TABLE book(id INT, price INT, code VARCHAR(50), PRIMARY KEY(id));`,
+			want: "ALTER TABLE `book` ADD COLUMN (`price` INT);\nALTER TABLE `book` ADD COLUMN (`code` VARCHAR(50));\n",
```

Comment:
The best way is to merge the multi-ALTER TABLE statement into one ALTER TABLE statement with multi-specs.

Reply:
Done.